src/components/Map.js

```diff
@@ -0,0 +1,396 @@
+        .selectAll("path")
+        .data(data.features)
+        .enter()
+        .append("path")
+        .attr("class", className)
+        .attr("d", geoPathGenerator)
+        .attr("stroke",stroke)
+        .attr("stroke-width", strokeWidth)
+        .attr("fill", fill)
+        .attr("fill-opacity", fillOpacity);
+
+    return path;
+}
+
+// Draw path
+export function drawPath(chartId, data, className, stroke = "#D7D7D7", strokeWidth = 1, strokeOpacity = .5, fill ="none", fillOpacity) {
+
+    let geoPathGenerator = d3.geoPath().projection(projection);
+
+    let path = d3.select(`#${chartId} svg`)
+        .append("g")
+        .selectAll("path")
+        .data(data)
+        .enter()
+        .append("path")
+        .attr("class", className)
+        .attr("d", geoPathGenerator)
+        .attr("stroke", stroke)
+        .attr("stroke-width", strokeWidth)
+        .attr("stroke-opacity", strokeOpacity)
+        .attr("fill-opacity", fillOpacity)
+        .attr("fill", fill);
+
+    return path;
+}
+
+// Create households
+export function createHouses(chartId, data, className = "household") {
+
+    // console.log(data)
+    // let points = g
+    // .append("g")
+
+
+    let points = d3.select(`#${chartId} svg`)
+                .append("g")
+    
+    points
+        .selectAll("circle")
+        .data(data)
+        .enter()
+        .append("circle")
+            .attr("class", className)
+            .attr("cx", function(d) {return projection([d.properties.x, d.properties.y])[0];})
+            .attr("cy", function(d) {return projection([d.properties.x, d.properties.y])[1];})
+            .attr("r", 1)
+            .attr("fill", "#36479D")
+            .attr("fill-opacity", .5)
+            .attr("stroke", "#36479D")
+            .attr("stroke-opacity", 1);
+
+    // let tw = g.node().clientWidth;
+    // let th = g.node().clientHeight;
+    // let sx = tw / paramsMap.width;
+    // let sy = th / paramsMap.height;
+
+    // let tooltip = d3.select(`#tooltip`)
+    //     .append("div")
+    //     .attr("class", "tooltip");
+    
+    // points.on("mouseover", function(e, d) {
+
+    //     console.log(d)
+    //     let x = sx*(+d3.select(this).attr("cx")) + 20;
+    //     let y = sy*(+d3.select(this).attr("cy")) - 10;
+
+    //     tooltip.style("visibility", "visible")
+    //         .style("top", `${y}px`)
+    //         .style("left", `${x}px`)
+    //         // .html(`${d.properties.place}`);
+
+    // }).on("mouseout", function() {
+    //     tooltip.style("visibility", "hidden");
+    // });
+
+    return points;
+}
+
+// https://stackoverflow.com/questions/13455042/random-number-between-negative-and-positive-value
+function getNonZeroRandomNumber(min, max) {
+    var random = Math.floor(Math.random()*min) - max;
+    if(random==0) return getNonZeroRandomNumber();
+    return random;
+}
+
+// Updates the households
+//Adapted from http://bl.ocks.org/JMStewart/6455921
+export function updateHouses(data) {
```

Comment:
@dfarooq610 There is a bug here so the points aren't animating. I wasn't able to figure it out tonight ... too tired. Maybe you'll have more luck?